lib/site_prism/dsl.rb

```diff
@@ -205,13 +205,19 @@ def create_helper_method(proposed_method_name, *find_args)
         end
       end
 
-      def create_rspec_existence_matchers(element_name)
+      def create_rspec_existence_matchers(element_name) # rubocop:disable Metrics/MethodLength
```

Comment:
Can we absorb this into the todo file (I know it's old, I'm coming into some time soon to fix these up)

Reply:
I'm not sure I can localize the disabling quite as specifically in the todo file, but if you're ok with me either disabling that cop for this file, or changing the global method length limit, that's easy enough

Reply:
Alter the limit value, that's the best approach